docs/reference/migration/migrate_7_10.asciidoc

```diff
@@ -144,6 +145,22 @@ the old behavior is supported until the next major release.
 To find out if you are using any deprecated functionality,
 enable <<deprecation-logging, deprecation logging>>.
 
+[discrete]
+[[breaking_710_cluster_deprecations]]
+==== Cluster deprecations
+
+[[deprecate-cluster-join-timeout]]
+.The `cluster.join.timeout` setting is deprecated.
+[%collapsible]
+====
+*Details* +
+The `cluster.join.timeout` node setting is now deprecated. In 7.x clusters, join
```

Comment:
Should we add that it's removed in the next major release? Or is that implied by the fact that it's deprecated?